Automattic/jetpack
# MU WPCOM: Port jetpack-global-styles from ETK

Fixes https://github.com/Automattic/dotcom-forge/issues/8057

## Proposed changes:

This PR ports jetpack-global-styles from ETK, as discussed in pbxlJb-5Xo-p2.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Patch this PR
* Activate one of themes in https://wordpress.com/themes/all/filter/editor-fonts (e.g., Leven)
* Go to page or post editor
	* Change font selection
	* Publish fonts
		<img width="288" alt="Screenshot 2024-07-05 at 17 17 06" src="https://github.com/Automattic/jetpack/assets/5287479/38cd5064-1a68-40f4-a3f1-08f32c36be21">
	* Observe fonts are applied on frontend
* Go to Appearance > Customize
	* Click "Fonts" section
	* Observe the links. ⚠️ Note that the first link is displayed only if your "Your homepage displays" is set to a static page.
		<img width="458" alt="Screenshot 2024-07-05 at 17 17 48" src="https://github.com/Automattic/jetpack/assets/5287479/bec52ee8-838e-4618-a834-3dfb59a86e80">
